src/main/webapp/style.css

```diff
@@ -69,3 +65,39 @@ input[type=submit]:hover{
 	display: inline-block; 
 	vertical-align: middle;  
 }
+
+
+/* Animation and Background colors */
+.background {
+  animation:slide 3s ease-in-out infinite alternate;
+  background-image: linear-gradient(-60deg, darkgrey 50%, black
```

Comment:
can this fit in one line?

Reply:
Yes, I'll fix it... prob was an accidental enter key haha